tokens/properties/mixins/InlineProgressBar.json

```diff
@@ -0,0 +1,39 @@
+{
+    "wikit-InlineProgressBar": {
+        "background-color": {
+            "value": "{background.color.progressive.default.value}"
+        },
+        "height": {
+            "value": "{dimension.static.size.50.value}"
+        },
+        "animation": {
```

Comment:
I don't think we need to create tokens for the animations' timing-functions. After all, if WiKit were to continue, it would be ideal to get rid of the tokens that document CSS values, and use mixins  (with duration tokens) to apply transitions and animations instead.